src/models/generated/ColocatedSupplyStorageTechnology.jl

```diff
@@ -0,0 +1,361 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
+    mutable struct ColocatedSupplyStorageTechnology{T <: PSY.Generator} <: Technology
+        base_power::Float64
+        lifetime_storage::Int
+        available::Bool
+        operation_costs_solar::PSY.OperationalCost
+        capacity_limits_wind::MinMax
+        name::String
+        capital_costs_power::PSY.ValueCurve
+        capacity_power_limits::MinMax
+        lifetime_wind::Int
+        capacity_energy_limits::MinMax
+        operations_costs_power::PSY.OperationalCost
+        duration_limits::MinMax
+        id::Int64
+        existing_capacity_solar::Float64
+        capital_costs_energy::PSY.ValueCurve
+        min_inverter_capacity::Float64
+        operation_costs_inverter::PSY.OperationalCost
+        inverter_efficiency::Float64
+        existing_capacity_wind::Float64
+        financial_data_solar::TechnologyFinancialData
+        power_systems_type::String
+        capacity_limits_solar::MinMax
+        financial_data_wind::TechnologyFinancialData
+        existing_capacity_power::Float64
+        internal::InfrastructureSystemsInternal
+        operation_costs_wind::PSY.OperationalCost
+        balancing_topology::String
+        efficiency_storage::InOut
+        region::Vector{Region}
+        losses_storage::Float64
+        inverter_supply_ratio::Float64
+        ext::Dict
+        capital_costs_wind::PSY.ValueCurve
+        lifetime_solar::Int
+        base_year::Int
+        existing_capacity_energy::Float64
+        capital_costs_inverter::PSY.ValueCurve
+        max_inverter_capacity::Float64
+        operations_costs_energy::PSY.OperationalCost
+        financial_data_storage::TechnologyFinancialData
```

Comment:
The Colocated should have a single `financial_data`. It's a single project, so we should remove `financial_data_storage`, `financial_data_solar`, `financial_data_wind` and simply leave a `financial_data` field.